tests/lib/rules/no-invalid-attribute-name.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * @author *****your name*****
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const RuleTester = require('eslint').RuleTester
+const rule = require('../../../lib/rules/no-invalid-attribute-name')
+
+const tester = new RuleTester({
+  parser: require.resolve('vue-eslint-parser'),
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module'
+  }
+})
+
+tester.run('no-invalid-attribute-name', rule, {
+  valid: [
+    {
+      filename: 'test.vue',
+      code: `
+      <template>
+        <div>
+          <p foo>
+            {{ content }}
+          </p>
+        </div>
+      </template>
+      `
```

Comment:
Please simplify all test cases to something like this, since the content is not important to this rule:

```suggestion
      code: '<template><p foo /></template>'
```